code/__DEFINES/jobs.dm

```diff
@@ -39,19 +39,19 @@
 #define CURATOR			(1<<5)
 #define QUARTERMASTER	(1<<6)
 #define CARGOTECH		(1<<7)
-#define MINER			(1<<8)
-#define LAWYER			(1<<9)
-#define CHAPLAIN		(1<<10)
-#define CLOWN			(1<<11)
-#define MIME			(1<<12)
-#define ASSISTANT		(1<<13)
-#define GIMMICK 		(1<<14)
-#define BARBER		    (1<<15)
-#define MAGICIAN        (1<<16)
-#define HOBO            (1<<17)
-#define SHRINK          (1<<18)
-#define CELEBRITY       (1<<19)
-#define MAILMAN			(1<<20) //MonkeStation Edit: Mailman Gimmick
+#define MAILMAN			(1<<8)
+#define MINER			(1<<9)
+#define LAWYER			(1<<10)
+#define CHAPLAIN		(1<<11)
+#define CLOWN			(1<<12)
+#define MIME			(1<<13)
+#define ASSISTANT		(1<<14)
+#define GIMMICK 		(1<<15)
+#define BARBER		    (1<<16)
+#define MAGICIAN        (1<<17)
+#define HOBO            (1<<18)
+#define SHRINK          (1<<19)
+#define CELEBRITY       (1<<20)
```

Comment:
#define JOB_DISPLAY_ORDER is what you need to add mailman to, not this one

Reply:
okay, where is that?

Reply:
revert changes on that, scroll down until you find #define JOB_DISPLAY_ORDER

Reply:
does that look good?

Reply:
frick i accidentally deleted the whole file

Reply:
oh god i'd have to remake the whole file, is it okay if i try again with a new PR and branch??